<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Location</class>
 <widget class="QDialog" name="Location">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>182</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_addr">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_addr"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_ward">
        <property name="text">
         <string>Ward    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_ward"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_district">
        <property name="text">
         <string>District  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_district"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_city">
        <property name="text">
         <string>City       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_city"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="button_save">
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
